tests/comfy_cli/command/test_workflow_notes.py

```
@@ -0,0 +1,547 @@
+        assert env["ok"] is False
+        assert env["error"]["code"] == "workflow_not_frontend_format"
+
+    def test_rejects_missing_file(self, tmp_path, capsys):
+        missing = str(tmp_path / "nope.json")
+        _force_json_renderer()
+        assert _invoke(["notes", missing]).exit_code == 1
+        env = _run(["notes", missing], capsys)
+        assert env["ok"] is False
+        assert env["error"]["code"] == "workflow_not_found"
+
+    def test_rejects_invalid_json(self, tmp_path, capsys):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)